plugins/module_utils/job.py

```diff
@@ -90,6 +130,7 @@ def _parse_jobs(output_str):
 
     Returns:
         list[dict]: A list of jobs and their attributes.
+        If no job status is found, this will return an empty job code with msg=Job not found
```

Comment:
See earlier comment about the msg "Job not found" not being equal to "Job Not Found"

Reply:
yep